web/src/main/webapp/loadMenu.ts

```diff
@@ -42,13 +42,15 @@ export class LoadMenu extends RemoteObject implements IDataView {
     private loadMenu: SubMenu;
     public readonly viewKind: ViewKind;
 
-    constructor(protected init: InitialObject, protected page: FullPage) {
+    constructor(protected init: InitialObject, protected page: FullPage, protected bookmarkFile: string) {
         super(init.remoteObjectId);
         this.viewKind = "Load";
         this.top = document.createElement("div");
         this.console = new ErrorDisplay();
         this.top.appendChild(this.console.getHTMLRepresentation());
         this.getUIConfig();
+        // Check whether the user is trying to visit a bookmark link
```

Comment:
this is actually a small race, because both getUIConfig and openingBookmark will execute asynchronously, but it will probably work.